packages/containers/src/List/List.sagas.js

```diff
@@ -0,0 +1,80 @@
+import { put, takeEvery, call } from 'redux-saga/effects';
+import { List } from 'immutable';
+import Connected from './List.connect';
+import Constants from './List.constant';
+import { getCollectionItems, sortCurried, filterList } from './utils';
+
+const getFilteredItems = filterListFn => (componentState, config, results) => {
+	const filterItemsCurried = sortCurried(componentState, config);
+	return filterItemsCurried(filterListFn(componentState, config, results));
+};
+
+export function* updateList({ props, context }) {
+	const collectionId = props.collectionId || 'default';
+	const state = context.store.getState();
+	const stateItems = getCollectionItems(state, collectionId);
+	const componentState = state.cmf.components.getIn(['Container(List)', collectionId]);
+
```

Comment:
Extract here in `const` the `isAsc`, `sortBy`, `searchQuery` and `columnsConfig`.
It's here you need to deal if the componentState doesn't exists or for those values with default or something else.
I don't see a case where component state don't exists other that selecting a bad part of the state, so it should always exists.